addons/web_tour/static/src/tour_service/tour_utils.js

```diff
@@ -1,9 +1,17 @@
 /** @odoo-module **/
 
 import { markup } from "@odoo/owl";
-import { isMobileOS } from "@web/core/browser/feature_detection";
 import { _t } from "@web/core/l10n/translation";
 import { _legacyIsVisible } from "@web/core/utils/ui";
+import { uiService, SIZES } from "@web/core/ui/ui_service";
+
+/* Recreating `ui.isSmall`
+ * This sucks.
+ * TODO: find an alternative solution
```

Comment:
maybe define isSmall next to getSize? I don't know why getSize is there actually. But both could be simple functions exported by the ui service file, and the service could use them directly

Reply:
Actually it's the way I did it at first, but then realized that in that case we lose the "memoization" of the size as `ui.size` is currently only updated on `resize` event, instead of evaluated at each call.

We could extract this `size` variable outside the `uiService`... but not sure if it's what we want :thinking: